### PR TITLE
D8CORE-4348 Setup Gitpod

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Setup gitpod configurations.

# Need Review By (Date)
- :shrug: 

# Urgency
- low

# Steps to Test
1. follow the instructions in the Readme file.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -1 +1,77 @@
...
+      [[ ! -z $SSH_PUBLIC_KEY  ]] &&
+      echo $SSH_PUBLIC_KEY | base64 -d > ~/.ssh/id_rsa.pub &&
+      chmod 644 ~/.ssh/id_rsa.pub &&
+      [[ ! -z $SSH_PRIVATE_KEY  ]] &&
+      echo $SSH_PRIVATE_KEY | base64 -d > ~/.ssh/id_rsa &&
+      chmod 600 ~/.ssh/id_rsa &&
+      [[ ! -z $GITCONFIG  ]] &&
+      echo $GITCONFIG | base64 -d > ~/.gitconfig &&
+      chmod 644 ~/.gitconfig
+    init: >
```
Comment (imonroe) [inline .gitpod.yml:38]: Should this be: 
```suggestion
  - init: >
```
?  

The docs seem to indicate that `before` `init` and `coomand` are the same sort of thing, and it seems like they should have the same yml?

Comment (imonroe) [reply .gitpod.yml:38]: Maybe same with `command` below?

**Review by imonroe — APPROVED** [review]
This works for me and I can get it launched in Gitpod correctly.

File: .gitpod.yml (hunk 1 above)
Comment (pookmish) [reply .gitpod.yml:38]: Good question, but no. Basically the `tasks` is an array of arrays. Look at it in the way of a php array
```
'tasks' => [
  [
    'before' => ...
    'init' => ...
    'command' => ...
  ],
  [
    'before' => ...
    'command' => ...
  ],
]
```
those three are treated as the same task. adding the `-` would then treat them as separate tasks. see the first example here: https://www.gitpod.io/docs/references/gitpod-yml#tasks
> Each array element opens in its own terminal.

We could do things in parallel like initiate drupal and spin up a decoupled server directory at the same time with 2 different terminals. But we want things to happen sequentially for this implementation.

Comment (imonroe) [reply .gitpod.yml:38]: Gotcha.  thanks for the clarification.

**Pull request merged**